app/portainer/components/forms/git-form/git-form-auth-fieldset/index.js

```diff
@@ -6,6 +6,7 @@ export const gitFormAuthFieldset = {
   bindings: {
     model: '<',
     onChange: '<',
+    showAuthExplanation: '<',
```

Comment:
this proprty is not used, right? or was it missing?

Reply:
It is used, please double check.

Reply:
I just don't see the change in the template, so I guess it was missing?

Reply:
Oh I see what you mean and yes.